src/main/java/ProjektiPizza/demo/security/JwtSuccessHandler.java

```diff
@@ -0,0 +1,15 @@
+package ProjektiPizza.demo.security;
+
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class JwtSuccessHandler implements AuthenticationSuccessHandler {
+    @Override
+    public void onAuthenticationSuccess(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, Authentication authentication) {
+        System.out.println("Successfully Authentication");
```

Comment:
Can we use a logger here instead of printing with the system.out?

Reply:
Yes we can.